ddapm_test_agent/agent.py

```diff
@@ -215,6 +215,20 @@ async def handle_v06_tracestats(self, request: Request) -> web.Response:
         )
         return web.HTTPOk()
 
+    async def handle_info(self, request: Request) -> web.Response:
+        return web.json_response(
```

Comment:
hmm, weird to have this as a mock?

Reply:
🤷‍♂️ I think I'm ok with it, ensures that it doesn't change unexpectedly

Reply:
mmm it's going to be fundamentally different since the agents support different features